src/embed_tests/TestOperator.cs

```diff
@@ -23,6 +24,120 @@ public void Dispose()
             PythonEngine.Shutdown();
         }
 
+        // Mock Integer class to test math ops on non-native dotnet types
+        public struct OwnInt
```

Comment:
I know this is test code, but please override `Equals` and `GetHashCode` to clear the warning.